components/Modal/Schedule/ScheduleModal.module.scss

```
@@ -0,0 +1,91 @@
+.container {
+  width: 32rem;
+  padding: 1.5rem;
+  background-color: $white;
+  transform: translate(-50%, -50%);
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  border-radius: 16px;
+
+  .create-schedule-form {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 1rem;
+    align-self: stretch;
+
+    .schedule-name {
+      display: flex;
+      padding: .5rem 0rem;
+      align-items: center;
+      gap: .25rem;
+      align-self: stretch;
+      outline: none;
+      @include heading4-medium;
+
+      &::placeholder {
+        color: $gray600;
+      }
+    }
+  }
+
+  .time-location-container {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+
+    .time-container,
+    .location-container {
+      display: flex;
+      align-items: center;
+      gap: 1rem;
+      align-self: stretch;
+
+      p {
+        width: 2.5rem;
+        @include headline2-bold;
+
+      }
+
+      .time {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        gap: .125rem;
+
+
+        .start,
+        .end {
+          display: flex;
+          align-items: center;
+          gap: .3125rem;
+
+          .date {
+            @include headline2-bold;
+          }
+
+          .time {
+            @include headline2-bold;
+          }
+        }
+      }
+
+      .location {
+        width: 100%;
+        display: flex;
+        padding: 8px 4px;
+        align-items: center;
+        gap: 4px;
```

```suggestion
        padding: 0.5rem 0.25rem;
        align-items: center;
        gap: 0.25rem;
```
p4: 여기 px이 있는 것 같아요!